ggmolvis/compositor.py

```diff
@@ -29,3 +33,37 @@ def _set_compositor_bg(rgba: tuple[float, float, float, float]):
     links.new(render_layers.outputs['Alpha'], mix_node.inputs[0])
     links.new(render_layers.outputs['Image'], mix_node.inputs[2])
     links.new(mix_node.outputs[0], composite.inputs[0])
+
+
+def _create_frame_image(frame_number: int,
+                        width: int,
+                        height: int,
+                        text: str):
+    img = Image.new('RGBA', (width, height), (0, 0, 0, 0))
+    draw = ImageDraw.Draw(img)
+    font = ImageFont.truetype("/System/Library/Fonts/Geneva.ttf", 80)
+    text_bbox = draw.textbbox((0, 0), text, font=font)
+    text_width = text_bbox[2] - text_bbox[0]
+    text_height = text_bbox[3] - text_bbox[1]
+    x = (width - text_width) // 2
+    y = height - text_height - 40
+    draw.text((x, y), text, font=font, fill=(255, 255, 255, 255))
```

Comment:
would probably need a way for user to specify the color of the font, especially if they are also specifying the color of the background